### DISCUSSION TITLE
Default and Required in go-pdk Schema

### DISCUSSION THREAD

**Discussion by segara2410** [Kong/kong#8261] [Help]
I am building a custom plugin using Golang but I don't find a way to set a config required or set its default value. Any thoughts?

**Comment by javierguerragiraldez** [comment]
Currently there's no way to define that from the Go side, unfortunately.  What we've been doing is the common Go style of "zero value is a valid state"

**Reply by segara2410** [reply]
Is there a way to prevent kong from running when there is any required config missing in my custom plugin?
